sdk/communication/Azure.Communication.CallAutomation/src/CallAutomationClient.cs

```diff
@@ -207,8 +207,24 @@ public virtual Response<AnswerCallResult> AnswerCall(AnswerCallOptions options,
 
         private AnswerCallRequestInternal CreateAnswerCallRequest(AnswerCallOptions options)
         {
+            // validate callbackUri
```

Comment:
There was feedback we should not do any server parameter from client side - see guideline here: https://azure.github.io/azure-sdk/dotnet_introduction.html#parameter-validation

GA version does this (main branch should probably follow this also)

Reply:
Applies to all validation this PR includes

Reply:
Made the suggested change, thanks!